SPECS/rpm-ostree/rpm-ostree.spec

```diff
@@ -8,44 +8,72 @@ URL:            https://github.com/cgwalters/rpm-ostree
 Vendor:		VMware, Inc.
 Distribution:	Photon
 # We always run autogen.sh
-BuildRequires: autoconf automake libtool
+BuildRequires: autoconf automake libtool git
 BuildRequires: json-glib-devel
+BuildRequires: gtk-doc
 BuildRequires: libcap
-BuildRequires: ostree
+BuildRequires: ostree-devel
 BuildRequires: libgsystem
-BuildRequires: rpm-devel
-BuildRequires: hawkey-devel
 BuildRequires: docbook-xsl
 BuildRequires:	libxslt
 BuildRequires:	gobject-introspection-devel
+BuildRequires:	openssl-devel
+BuildRequires:	libhif-devel >= 0.2.0
+BuildRequires: 	hawkey-devel >= 0.4.6
+BuildRequires: 	rpm-devel >= 4.11.0
+BuildRequires: 	librepo-devel >= 1.7.11
+BuildRequires:	attr
+BuildRequires: 	python2-libs
+BuildRequires:	python2
+BuildRequires: 	gobject-introspection-python
+
 Requires:      yum
 BuildRequires:	which
 BuildRequires:	popt-devel
 Requires:	libcap
+Requires:	librepo
 Requires:	hawkey
+Requires:	rpm
+Requires:	libhif
 Requires:	ostree
 Requires:	json-glib
 
 
-
 %description
 This tool takes a set of packages, and commits them to an OSTree
 repository.  At the moment, it is intended for use on build servers.
 
+%package devel
+Summary: Development headers for rpm-ostree
+Group: Development/Libraries
+Requires: %{name} = %{version}-%{release}
+
+%description devel
+Includes the header files for the rpm-ostree library.
+
 %prep
-%setup -q -n %{name}-%{version}
+%autosetup -Sgit -n %{name}-%{version}
+git clone git://git.gnome.org/libglnx libglnx
 
 %build
 env NOCONFIGURE=1 ./autogen.sh
-%configure --disable-silent-rules --enable-patched-hawkey-and-libsolv --enable-usrbinatomic
+%configure --disable-silent-rules --enable-usrbinatomic
```

Comment:
This is being reported as an invalid configuration option now. rpm-ostree does not look like needs patched versions anymore.